src/pages/index.astro

```diff
@@ -41,6 +45,86 @@ import "../index.scss";
+          vision.
+        </p>
+        <div class="mt-24">
+          <ul class="grid md:grid-cols-2 gap-8 md:gap-16">
+            <li class="flex justify-stretch">
+              <CardProject
+                img={{
+                  classProp: "rounded-3xl w-64 h-64",
+                  src: "astro-lp/project_1.png",
+                  alt: "Weather App Layout",
+                }}
+                title="'Weather App"
+                description=`Stay ahead of the forecast with our intuitive weather app. Get real-time updates on current conditions, hourly forecasts, and daily outlooks. Plan your day confidently with accurate weather information at your fingertips. Never be caught off guard by the elements again!`
+                skills={["HTML", "CSS", "JavaScript"]}
+              />
+            </li>
+
+            <li class="flex justify-stretch">
+              <CardProject
+                img={{
+                  classProp: "rounded-3xl w-64 h-64",
+                  src: "astro-lp/project_2.png",
+                  alt: "Age calculator Layout",
+                }}
+                title="Age calculator"
+                description=`AgeEase: Instantly calculate your age or anyone else' s with precision and simplicity. Whether you're celebrating milestones or planning events, keeps track of the years, months, and days. Effortlessly determine ages for birthdays, anniversaries, or any special occasion.`
+                skills={["HTML", "CSS", "JavaScript"]}
+              />
+            </li>
+            <li class="flex justify-stretch">
+              <CardProject
+                img={{
+                  classProp: "rounded-3xl w-64 h-64",
+                  src: "astro-lp/project_3.png",
+                  alt: "Real Estate welcome page",
+                }}
+                title="Real Estate"
+                description=`Introducing Real Estate: Your trusted partner in real estate management. From residential complexes to commercial spaces, we offer comprehensive property management services tailored to meet your needs. With a focus on efficiency, transparency, and maximizing property value.`
+                skills={["HTML", "CSS", "JavaScript"]}
+              />
+            </li>
+            <li class="flex justify-stretch">
+              <CardProject
+                img={{
+                  classProp: "rounded-3xl w-64 h-64",
+                  src: "astro-lp/project_4.png",
+                  alt: "Forest Group welcome page",
+                }}
+                title="Forest Group"
+                description=`Welcome to Forest Group: Your digital oasis for landscaped parks. Explore lush green spaces, tranquil gardens, and vibrant landscapes from the comfort of your screen. Discover inspiration for your next outdoor adventure, family picnic, or leisurely stroll.`
+                skills={["HTML", "CSS", "JavaScript"]}
+              />
+            </li>
```

Comment:
This is not dry, use `.map` to fix it :)

Reply:
Changed